src/main/java/com/geel/hunterrumours/HunterRumoursPlugin.java

```diff
@@ -99,6 +99,10 @@ protected void startUp() throws Exception
 	protected void shutDown() throws Exception
 	{
 		overlayManager.remove(overlay);
+		if (infoBox != null)
```

Comment:
fyi it is safe to do `infoBoxManager.removeInfoBox(null)` so this guard isn't strictly needed.

Reply:
Thanks for letting me know!